solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -963,23 +965,6 @@ contract LiquidityProtection is ILiquidityProtection, TokenHandler, Utils, Owned
         return Fraction({ n: reserveBalance.mul(2), d: poolTokenSupply });
     }
 
-    /**
-     * @dev returns the average rate of 1 reserve token in the other reserve token units
-     *
-     * @param _poolToken            pool token
-     * @param _reserveToken         reserve token
-     * @param _validateAverageRate  true to validate the average rate; false otherwise
-     */
-    function reserveTokenAverageRate(
```

Comment:
average rate should still be used when calculating the IL (not when calculating the fees).